packages.json

```diff
@@ -34,20 +34,11 @@
             "all": [
                 "firefox",
                 "firefox-langpacks",
+	        "gnome-software-rpm-ostree", 
                 "gnome-tour"
             ]
         }
     },
-    "37": {
-        "include": {
-            "all": [
-	        "podman-docker"
```

Comment:
I must be missing something.  How does `podman-docker` relate to Gnome Software?

I understand why we're removing it from all images - because we're going to install the UBlue version, but what was the issue with podman-docker?

Reply:
Oh, F37 builds are disabled anyway.  This has no effect.

LGTM